src/java/voldemort/utils/pool/KeyedResourcePool.java

```diff
@@ -557,9 +557,10 @@ private void throwReportedExceptions() throws Exception {
                 skippedExceptionCount++;
                 if(elapsedTime <= excpetionReportTimeMS) {
                     Exception e = entry.getSecond();
-                    logger.info(" Throwing remembered exception. time elapsed (ms) " + elapsedTime
-                                + ". Exception : "
-                                + e.getMessage());
+                    if(logger.isDebugEnabled()) {
```

Comment:
From the name of this class, it could be used by multiple places, so is it possible that we still need info logging in other scenarios?